File: Go言語/基礎知識用の勉強/1変数勉強/実行済み/データ型.go
package main

import "fmt"

func main() {
	asakuno := "朝来野智博"
	var tomohiro string = "asakuno tomohiro"

	fmt.Println(asakuno, len(asakuno))
	fmt.Println(tomohiro, len(tomohiro))

	fmt.Println(asakuno, asakuno[0])           // 朝来野智博 230
	fmt.Println(tomohiro, tomohiro[0])         // asakuno tomohiro 97
	fmt.Println(tomohiro, string(tomohiro[0])) // asakuno tomohiro a
}

/* vim: set ts=4 sts=4 sw=4 tw=0 ff=unix fenc=utf-8 ft=go noexpandtab: */

File: Java言語/基礎知識用の勉強/1変数勉強/データ型.java
class データ型 {
	public static void main(String[] args){
		int asakuno = 20210821;
//		float tomohiro = 2021.08;	// error| エラー: 不適合な型: 精度が失われる可能性があるdoubleからfloatへの変換
		float tomohiro = 2021.08f;
		String asakunotomohiro = "朝来野智博";
		boolean asa = false;

		System.out.println(asakuno);	// 20210821
		System.out.println(tomohiro);	// 2021.08
		System.out.println(asakunotomohiro);	// 朝来野智博
		System.out.println(asa);	// false

		// 以下、数値を文字列として結合した。
		System.out.println(asakuno + "" + tomohiro);	// 202108212021.08
	}
}
